wandb/sdk/interface/interface.py

```diff
@@ -474,6 +474,28 @@ def _communicate_artifact(
     ) -> MessageFuture:
         raise NotImplementedError
 
+    def download_artifact(
```

Comment:
Nit: methods in this class are prefixed with the communication type. I suppose this should be `communicate_download_artifact`.